lib/src/log_printer.dart

```diff
@@ -8,7 +8,7 @@ part of logger;
 ///
 /// You can implement a `LogPrinter` from scratch or extend [PrettyPrinter].
 abstract class LogPrinter {
-  List<String> _buffer;
+  List<String> buffer;
```

Comment:
Prepending this field with an underscore communicates that it is a private field, which I think is correct. Not doing so does the opposite and lets users of this package twiddle with the value, which is a Bad Idea. I'd leave it. 

If anything I think this field should be removed since this field is used to store results of a (sequence of) method call(s). `println` could be replaced by having the `log` method return a `List<String>` (or perhaps a `String` with newlines).

Reply:
Hi! Thanks for your code review! Gonna try to find a solution to this when I have some time

Reply:
I gave this some thought too and the solution is actually quite simple. 
- remove the `_buffer` altogether
- change the signature of the `log` method to `List<String> log(LogEvent event)`
- on [line 115](https://github.com/leisim/logger/blob/28429d293ec79bc6dfa60c9419d66dcba9f40a33/lib/src/logger.dart#L115) capture `List<String>` and feed it to the `_output` there. 

If @leisim thinks this is a good idea I'll create the PR.